content/03.results.md

```diff
@@ -194,45 +193,79 @@ As expected, EXTEND scores significantly correlated with _TERC_ (R = 0.619, p <
+| PT_S0Q27J13               | BS_P3PF53V8               | 7316-2307 | Initial CNS Tumor | Solid Tissue      | Temozolomide, Irinotecan, Radiation              | None documented                       | 15.482649479451206 | HGG, H3 wildtype, TP53 activated |
+| PT_VTM2STE3               | BS_ERFMPQN3               | 7316-2189 | Progressive       | Derived Cell Line | Unknown                                          | Lynch Syndrome                        | 5.739499354950266  | HGG, H3 wildtype, TP53 loss      |
+| PT_VTM2STE3               | BS_02YBZSBY               | 7316-2189 | Progressive       | Solid Tissue      | Unknown                                          | Lynch Syndrome                        | 274.48805695576783 | HGG, H3 wildtype, TP53 activated |
 
-Table 2. Patients with hypermutant tumors. {#tbl:T2 tag="2"}	
 
+Table 2. Patients with hypermutant tumors.	
 
-![**Figure 4. _TP53_ and telomerase activity** A, Receiver Operating Characteristic for _TP53_ classifier run on FPKM of stranded RNA-Seq samples. B, Violin and box plots of _TP53_ scores plotted by _TP53_ alteration type (N<sub>activated</sub> = 27, N<sub>lost</sub> = 140, N<sub>other</sub> = 999). C, Violin and box plots of _TP53_ RNA expression plotted by _TP53_ activation status (N<sub>activated</sub> = 11, N<sub>lost</sub> = 96, N<sub>other</sub> = 862). D, Box plots of _TP53_ and telomerase (EXTEND) scores grouped by `cancer_group`. Mutation status is highlighted in orange (hypermutant) or red (ultra-hypermutant). E, Heatmap of RefSig mutational signatures for patients who have least one tumor with a TMB >= 10 Mut/Mb.](images/main/Figure4.png){#fig:Fig4}
+Next, we asked whether transcriptomic classification of _TP53_ dysregulation and/or telomerase activity recapitulate the known prognostic influence of these oncogenic biomarkers.
+We used an additive model to perform multivariate cox regression on overall survival (**STAR Methods**), controlling for extent of tumor resection and whether a tumor was low-grade (LGG group) or high-grade (HGG group). 
+Depicted in the forest plot (**Figure {@fig:Fig4}F**), we show an expected significant overall survival benefit if the tumor was fully resected (HR = 0.36, CI = 0.2-0.67, p = 0.001) or if the tumor belongs to the LGG group (HR = 0.068, CI = 0.0091-0.5, p = 0.008).
+In contrast, we show an expected significant risk if the tumor belongs to the HGG group (HR = 5.7, CI = 3.5-9.0, p < 0.001).
+High telomerase scores were a poor prognostic indicator across brain tumor histologies (HR = 24, CI = 7.1-79, p < 0.001), demonstrating that EXTEND scores calculated from RNA-Seq can be used as a rapid surrogate measure of telomerase activity.
+Higher _TP53_ scores, indicative of dysregulation of the _TP53_ gene or pathway, while not significant across the entire OpenPBTA cohort, resulted in a significant overall survival risk for both DMGs (HR = 1.77e6, CI = 1.98-1.57e12, p = 0.04) and ependymomas (HR = 1612, CI = 9.1-2.9e5, p = 0.005).
```

Comment:
@sjspielman this result is currently only in a notebook - I am not sure if you saw my note about possibly making a supp table 4 for survival results mentioned in the paper and then moving KR as supp 5 - thoughts? I can do that, if so.

Reply:
This makes sense to me! Go for it.